src/Open3D/GUI/Label.cpp

```diff
@@ -110,6 +118,10 @@ Widget::DrawResult Label::Draw(const DrawContext& context) {
     ImGui::SetCursorPos(
             ImVec2(frame.x - context.uiOffsetX, frame.y - context.uiOffsetY));
     ImGui::PushItemWidth(frame.width);
+    bool isDefaultColor = (impl_->color != DEFAULT_COLOR);
+    if (isDefaultColor) {
```

Comment:
I think there is misspell here, shouldn't it be `if (!isDefaultColor)`?

Reply:
Good point. I think I was trying to avoid inverted logic in the `if` but I can't think of a better name.